python/grass/pygrass/modules/grid/split.py

```diff
@@ -123,9 +122,10 @@ def split_region_tiles(region=None, width=100, height=100):
     nrows = (reg.rows + height - 1) // height
     box_list = []
     for row in range(nrows):
-        row_list = []
-        for col in range(ncols):
-            row_list.append(get_tile_start_end_row_col(reg, row, col, width, height))
+        row_list = [
```

Comment:
In the GUI code you solved similar situations with nested comprehensions, no?

Reply:
Even those using two times a "range()" (to create a number, that indexes the loop)?

Oh, I see why the tool only made one level. It looks like it is a list of lists, not one long list (if it was a list.extend). Probably reading the comprehension that is equivalent would be too ugly...